interface.js

```diff
@@ -47,6 +47,22 @@ window.addEventListener('DOMContentLoaded', () => {
 	updateSplitSizes(panes, splitInstance);
 });
 
+/* ========================================================================== */
+/* Dark Mode Listeners                                                        */
+/* ========================================================================== */
+
+// Get the initial value of isDarkMode from the store
+ipcRenderer.invoke('getStoreValue', 'isDarkMode').then((isDarkMode) => {
+    enabledProviders.forEach((provider) => provider.toggleDarkMode(isDarkMode));
```

Comment:
We should probably run this over all providers rather than just enabled ones so they don't get out of sync when people add/remove providers.

Reply:
Good idea, totally agree. We want the state for dark/light to stay in sync across all providers.